wolfcrypt/src/memory.c

```diff
@@ -1133,3 +1133,141 @@ void __attribute__((no_instrument_function))
 }
 #endif
 
+#ifdef WOLFSSL_LINUXKM_SIMD_X86_IRQ_ALLOWED
+union fpregs_state **wolfcrypt_irq_fpu_states = NULL;
+#endif
+
+#if defined(WOLFSSL_LINUXKM_SIMD_X86) && defined(WOLFSSL_LINUXKM_SIMD_X86_IRQ_ALLOWED)
```

Comment:
Please fix max line length in this section.

Reply:
done